README.md

```diff
@@ -1,4 +1,4 @@
-![SQLECTRON Logo](https://www.dropbox.com/s/038vmsywakz2hug/SQLECTRON-400px.png?raw=1)
+![SQLECTRON Logo](https://www.dropbox.com/s/dt06tif4xo1khbf/SQLECTRON-400px.png?raw=1)
```

Comment:
Yes. I am keeping it outside of the app just before the first release. To not increase the size of the git repository in case we change the images.